src/distribution.cpp

```diff
@@ -101,7 +101,7 @@ size_t DiscreteIndex::sample(uint64_t* seed) const
     return 0;
   }
 }
-
+assign
```

Comment:
I'm not sure if this line is intentional, but it definitely doesn't seem necessary.